### DISCUSSION TITLE
Game Server Reorganization

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#335] [Suggestions]
When dealing with Game Servers organization, is the request to allow for users to rearrange the display of servers and save them in a locked position. For example:
Ew, gross, servers all out of order. 
![image](https://user-images.githubusercontent.com/44717221/215411647-9b431b05-aa20-46e6-aa8b-1fe194bf7a86.png)

Ahh, yes, much better. 
![image](https://user-images.githubusercontent.com/44717221/215411788-03dfc847-b76a-4adb-90f5-b0b12190c311.png)

**Comment by mvaessen** [comment]
hi, in order for people to vote in suggestions it's important that each suggestion has a separate item. Otherwise people are forced to vote on both. Can you please move one of the suggestions to a separate item?

**Comment by Decollates** [comment]
Sorry about that!

**Comment by mvaessen** [comment]
Hi, this will be fixed in the upcoming update. Do note that the change involves inheriting the sort order from the cluster pages.
See this item for the sort order: https://github.com/gameserverapp/Platform/discussions/47
